server/supernova/sc/sc_osc_handler.cpp

```diff
@@ -1589,7 +1604,7 @@ void dump_controls(rt_string_stream & stream, abstract_synth const & synth, int
         } else
             stream << ", ";
 
-        stream << synth.get(control_index);
+        stream << synth.get(control_index); /*FIXME: this seems not to check for mapped controls*/
```

Comment:
AFAICT this seems to be the place where controls are printed, but omitting the mappings.